src/cookie.ts

```diff
@@ -7,10 +7,10 @@ export const createCookieManager = (event: MCEvent, settings: TrackerSettings) =
   const domainHash = getDomainHash(event.client.url.hostname + '/');
 
   return {
-    set: (name, value, opts?: ClientSetOptions) => {
+    set: (name, value, opts?: ClientSetOptions): void => {
       event.client.set(`${namespace}_${name}.${domainHash}`, value, opts);
     },
-    get: (name) => {
+    get: (name): string => {
       return event.client.get(`${namespace}_${name}.${domainHash}`);
```

Comment:
this might return `undefined` which clashes with the declared return type
```suggestion
      return event.client.get(`${namespace}_${name}.${domainHash}`) || '';
```